a10_octavia/controller/worker/flows/a10_member_flows.py

```diff
@@ -1006,3 +1007,272 @@ def get_batch_update_member_snat_pool_subflow(self, member):
+            batch_update_members_flow.add(vthunder_tasks.VThunderComputeConnectivityWait(
+                name=a10constants.BACKUP_CONNECTIVITY_WAIT,
+                requires=constants.AMPHORA,
+                rebind={a10constants.VTHUNDER: a10constants.BACKUP_VTHUNDER}))
+            batch_update_members_flow.add(vthunder_tasks.VCSSyncWait(
+                name="backup-plug-wait-vcs-ready",
+                requires=a10constants.VTHUNDER))
+            batch_update_members_flow.add(vthunder_tasks.GetMasterVThunder(
+                name=a10constants.GET_MASTER_VTHUNDER,
+                requires=a10constants.VTHUNDER,
+                provides=a10constants.VTHUNDER))
+            batch_update_members_flow.add(a10_database_tasks.GetBackupVThunderByLoadBalancer(
+                name="get_backup_vThunder_after_get_master",
+                requires=(constants.LOADBALANCER, a10constants.VTHUNDER),
+                provides=a10constants.BACKUP_VTHUNDER))
+            batch_update_members_flow.add(
+                vthunder_tasks.EnableInterfaceForMembers(
+                    name=a10constants.ENABLE_MASTER_VTHUNDER_INTERFACE,
+                    requires=[
+                        constants.ADDED_PORTS,
+                        constants.LOADBALANCER,
+                        a10constants.VTHUNDER]))
+            batch_update_members_flow.add(vthunder_tasks.AmphoraePostMemberNetworkPlug(
+                name="amphorae-post-member-network-plug-for-master",
+                requires=(constants.LOADBALANCER, constants.ADDED_PORTS,
+                          a10constants.VTHUNDER)))
+            batch_update_members_flow.add(
+                vthunder_tasks.VThunderComputeConnectivityWait(
+                    name=a10constants.CONNECTIVITY_WAIT_FOR_MASTER_VTHUNDER,
+                    requires=(a10constants.VTHUNDER, constants.AMPHORA)))
+            batch_update_members_flow.add(
+                vthunder_tasks.VThunderComputeConnectivityWait(
+                    name=a10constants.CONNECTIVITY_WAIT_FOR_BACKUP_VTHUNDER,
+                    rebind={
+                        a10constants.VTHUNDER: a10constants.BACKUP_VTHUNDER},
+                    requires=constants.AMPHORA))
+            batch_update_members_flow.add(vthunder_tasks.VCSSyncWait(
+                name="member_enable_interface_vcs_sync_wait",
+                requires=a10constants.VTHUNDER))
+            batch_update_members_flow.add(vthunder_tasks.GetMasterVThunder(
+                name=a10constants.GET_VTHUNDER_MASTER,
+                requires=a10constants.VTHUNDER,
+                provides=a10constants.VTHUNDER))
+            batch_update_members_flow.add(
+                vthunder_tasks.EnableInterfaceForMembers(
+                    requires=[
+                        constants.ADDED_PORTS,
+                        constants.LOADBALANCER,
+                        a10constants.VTHUNDER]))
+        batch_update_members_flow.add(database_tasks.MarkPoolActiveInDB(
```

Comment:
We do mark all the previous SLB components in ACTIVE state once we are done with the current operation, just to make sure that other components stays in ACTIVE state
 
Because create/update/delete  pool enters into PENDING_UPDATE state, so I added this task **MarkPoolActiveInDB**